packages/block-library/src/media-text/deprecated.js

```diff
@@ -0,0 +1,193 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { noop } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import {
+	InnerBlocks,
+	getColorClassName,
+} from '@wordpress/block-editor';
+
+/**
+ * Internal dependencies
+ */
+import { DEFAULT_MEDIA_WIDTH } from './index';
+
+export default [
+	{
```

Comment:
The deprecation was introduced at an earlier stage of the PR where it was needed. With the most recent changes to the PR, it might not be needed anymore. Thanks for pointing this out, I'll double check and address it accordingly.

Reply:
You were right, just pushed a commit that removes the deprecation. Good catch!